extruct/w3cmicrodata.py

```diff
@@ -18,11 +18,32 @@
     from urllib.parse import urljoin
 
 import lxml.etree
+from lxml.html.clean import Cleaner
 from w3lib.html import strip_html5_whitespace
+import html_text
 
 from extruct.utils import parse_html
 
 
+# Cleaner which is similar to html_text cleaner, but is less aggressive
```

Comment:
As far as I see, the difference between this cleaner and the html_text one is `embedded=False` and `frames=False`. It would be nice to include this in the comment and the reasoning about why. I imagine the reason is that we want to include frames and iframes content as well, right?